Gemfile

```diff
@@ -72,7 +71,8 @@ gem "defra_ruby_govpay"
 # With the environment properly configured, when any app in an environment needs
 # to call Companies House, instead it will call this app which will mock the end
 # point and return the response expected.
-gem "defra_ruby_mocks"
+gem "defra_ruby_mocks",
+    git: "https://github.com/DEFRA/defra-ruby-mocks.git"
```

Comment:
Why not use the gem version from rubygems.org?

Reply:
Let me publish the new version and then I will use that